ext/microprofile/mp-rest-client/src/main/java/org/glassfish/jersey/microprofile/restclient/RestClientBuilderImpl.java

```diff
@@ -142,15 +145,15 @@ public RestClientBuilder executorService(ExecutorService executor) {
             throw new IllegalStateException("Base uri/url cannot be null!");
         }
 
+        for (RestClientListener restClientListener : ServiceLoader.load(RestClientListener.class)) {
```

Comment:
Why not `ServiceFinder` instead of `ServiceLoader`?

Reply:
Changed.